curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,434 @@ Answer all of the questions below correctly to pass the quiz.
 #### --distractors--
 
-Placeholder distractor 1
+1.5:1
 
 ---
 
-Placeholder distractor 2
+7:1
 
 ---
 
-Placeholder distractor 3
+3:1
 
 #### --answer--
 
-Placeholder answer
+4.5:1
 
 ### --question--
 
 #### --text--
-
-Placeholder question
+What does the 'Inspect Element' feature do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Edits text directly
 
 ---
 
-Placeholder distractor 2
+Downloads files
 
 ---
 
-Placeholder distractor 3
+Modifies images
 
 #### --answer--
 
-Placeholder answer
+Displays CSS and HTML
 
 ### --question--
 
 #### --text--
-
-Placeholder question
+Which method keeps content in the document but hidden visually?
 
 #### --distractors--
 
-Placeholder distractor 1
+display: none;
 
 ---
 
-Placeholder distractor 2
+position: absolute;
 
 ---
 
-Placeholder distractor 3
+opacity: 0;
 
 #### --answer--
 
-Placeholder answer
+visibility: hidden;
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a tool for testing color combinations?
 
 #### --distractors--
 
-Placeholder distractor 1
+WordPress
 
 ---
 
-Placeholder distractor 2
+Sketch
 
 ---
 
-Placeholder distractor 3
+Excel
 
 #### --answer--
 
-Placeholder answer
+Paletton
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which browser is known for robust developer tools?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Internet Explorer
 
 ---
 
-Placeholder distractor 3
+Opera
 
 #### --answer--
 
-Placeholder answer
+Firefox
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common use for aria-hidden="true"?
 
 #### --distractors--
 
-Placeholder distractor 1
+To show content
 
 ---
 
-Placeholder distractor 2
+To display on mobile
 
 ---
 
-Placeholder distractor 3
+To increase visibility
 
 #### --answer--
 
-Placeholder answer
+To hide from screen readers
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which color is considered low contrast with white?
 
 #### --distractors--
 
-Placeholder distractor 1
+Black
 
 ---
 
-Placeholder distractor 2
+Dark Blue
 
 ---
 
-Placeholder distractor 3
+Dark Green
 
 #### --answer--
 
-Placeholder answer
+Light Gray
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool allows you to test CSS live on a website?
 
 #### --distractors--
 
-Placeholder distractor 1
+CodePen
 
 ---
 
-Placeholder distractor 2
+JSFiddle
 
 ---
-
-Placeholder distractor 3
+Live CSS Editor
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property makes an element invisible but still takes up space?
 
 #### --distractors--
 
-Placeholder distractor 1
+display: none;
 
 ---
 
-Placeholder distractor 2
+z-index: -1;
 
 ---
-
-Placeholder distractor 3
+opacity: 0;
 
 #### --answer--
 
-Placeholder answer
+visibility: hidden;
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the ideal contrast ratio for graphics?
```

Comment:
Again, WCAG does not define ideals, it defines the bare minimum for accessibility. Even `7:1` may be hard to read for some people. I do not think it is a good idea to assume there are "ideal" contrast ratios.